res/device-mqtt/configuration.toml

```diff
@@ -1,96 +1,130 @@
-# Copyright © 2020 Intel Corporation. All rights reserved.
-# SPDX-License-Identifier: BSD-3-Clause
+MaxEventSize = 0 # value 0 unlimit the maximum event size that can be sent to message bus or core-data
 
+[Writable]
+LogLevel = "INFO"
+  # InsecureSecrets are required for when Redis is used for message bus
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
+      [Writable.InsecureSecrets.DB.Secrets]
+      username = ""
+      password = ""
+    [Writable.InsecureSecrets.MQTT]
+    path = "credentials"
+      [Writable.InsecureSecrets.MQTT.Secrets]
+      username = ""
+      password = ""
+  [Writable.Reading]
+  ReadingUnits = true
+  
 [Service]
-Host = 'edgex-device-mqtt'
-Port = 48100
-BootTimeout = 30000
-Protocol = 'http'
-CheckInterval = '10s'
-ConnectRetries = 100
-ClientMonitor = 15000
-Labels = []
-StartupMsg = 'device simple started'
-Timeout = 5000
-EnableAsyncReadings = true
-AsyncBufferSize = 16
+HealthCheckInterval = "10s"
+Host = "localhost"
+Port = 59982
+ServerBindAddr = ""  # blank value defaults to Service.Host value
+StartupMsg = "device mqtt started"
+# MaxRequestSize limit the request body size in byte of put command
+MaxRequestSize = 0 # value 0 unlimit the request size.
+RequestTimeout = "5s"
 
 [Registry]
-Host = 'edgex-core-consul'
+Host = "localhost"
 Port = 8500
-Type = 'consul'
+Type = "consul"
 
 [Clients]
-  [Clients.Data]
-  Name = 'edgex-core-data'
-  Protocol = 'http'
-  Host = 'edgex-core-data'
-  Port = 48080
-  Timeout = 50000
+  [Clients.core-data]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 59880
+
+  [Clients.core-metadata]
+  Protocol = "http"
+  Host = "localhost"
+  Port = 59881
+
+[MessageQueue]
+Protocol = "redis"
+Host = "localhost"
+Port = 6379
+Type = "redis"
+AuthMode = "usernamepassword"  # required for redis messagebus (secure or insecure).
+SecretName = "redisdb"
```

Comment:
Interesting, this should not work as the other services are using MQTT for the MessageBus. Need to change for using mqtt